python/vpc-ec2-local-zones/app.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+import os
+
+import aws_cdk as cdk
+
+from vpc_ec2_local_zones.vpc_ec2_local_zones_stack import VpcEc2LocalZonesStack
+
+
+app = cdk.App()
+VpcEc2LocalZonesStack(app, "VpcEc2LocalZonesStack",
+    # If you don't specify 'env', this stack will be environment-agnostic.
+    # Account/Region-dependent features and context lookups will not work,
+    # but a single synthesized template can be deployed anywhere.
+
+    # Uncomment the next line to specialize this stack for the AWS Account
+    # and Region that are implied by the current CLI configuration.
+
+    env=cdk.Environment(account=os.getenv('CDK_DEFAULT_ACCOUNT'), region='us-east-1'),
```

Comment:
```suggestion
    env=cdk.Environment(account=os.getenv('CDK_DEFAULT_ACCOUNT'), region=os.getenv('CDK_DEFAULT_REGION', 'us-east-1')),
```

Maybe we should always first honor the default region defined in ~/.aws/config or env var otherwise fall back to `us-east-1`? This will be helpful if user is deploying local zone of other region.

Reply:
sounds good, I'm going to document it in the Readme